recipes/kafka_producer.rb

```diff
@@ -33,6 +33,15 @@
   notifies :restart, 'service[opennms]', :delayed
 end
 
+template "#{node['opennms']['conf']['home']}/etc/org.opennms.features.kafka.producer.client.cfg" do
+  source "#{template_dir}org.opennms.features.kafka.producer.client.cfg.erb"
+  owner 'root'
+  group 'root'
+  mode 00664
+  only_if { mv.to_i >= 26}
```

Comment:
this resource and the other templates you added would run on anything >= 26 but a template only exists in 26 - either change the guard or add more templates please